apps/api/src/schema/search.ts

```diff
@@ -7,7 +7,7 @@ export const searchQuerySchema = z.object({
   take: z.coerce
     .number()
     .default(100)
-    .refine((x) => x <= 100, "Page size must be smaller than 100"),
+    .refine((x) => x <= 100, "Page size must be less than or equal to 100"),
```

Comment:
There is a `.gte` built into Zod, which does what you want more concisely. It is equivalent and you do not have to use it; the current form is clear enough.